packages/components/src/globals/scss/_theme-tokens.scss

```
@@ -126,7 +126,7 @@ $button-padding-ghost-field: calc(0.675rem - 3px) 12px !default;
 /// @access public
 /// @group button
 /// Uses the same padding-y as small buttons, but removes extra padding-right
-$button-padding-ghost-sm: calc(0.375rem - 3px) 1rem !default;
+$button-padding-ghost-sm: calc(0.375rem - 3px) 12px !default;
```

```suggestion
$button-padding-ghost-sm: calc(0.375rem - 3px) rem(12px) !default;
```

Do we need the `rem` here potentially?

---

I am following the value for button padding above https://github.com/carbon-design-system/carbon/blob/035cd1105fb1bd8fa38aa28f73105b29f275ee9a/packages/components/src/globals/scss/_theme-tokens.scss#L96 but I can change them all if we want

---

@emyarod got it 👍 does it scale fine as text size grows? If so then definitely doesn't seem a problem as-is.

---

yeah I think so, I don't think we identified it as a problem during previous theming discussions, but I can change these values if we want to

---

Nah sounds good! Definitely down to keep the status quo 👍